Sming/Core/Network/Http/Websocket/WebsocketConnection.h

```diff
@@ -73,7 +73,6 @@ class WebsocketConnection
 
 	virtual ~WebsocketConnection()
 	{
-		state = eWSCS_Closed;
```

Comment:
What is the reason to remove the change of state here?

Reply:
if close method got connection in eWSCS_Closed state it do not fire onDisconnect delegate and this is not good. close() method change state by itself, and as far as I understand changing state to eWSCS_Closed in close() method do not hurt anything.

Reply:
https://github.com/SmingHub/Sming/blob/08c9eb18ac672f6e974d77885b82b56d972fe6f3/Sming/Core/Network/Http/Websocket/WebsocketConnection.cpp#L259 Here is it, on line 269 there is delegate call, but it inside if(state != eWSCS_Closed) ...